packages/two_dimensional_scrollables/lib/src/table_view/table_delegate.dart

```diff
@@ -16,12 +16,12 @@ import 'table_span.dart';
 /// [TableView].
 typedef TableSpanBuilder = TableSpan Function(int index);
 
-/// Signature for a function that creates a child [Widget] for a given
+/// Signature for a function that creates a child [TableViewCell] for a given
 /// [TableVicinity] in a [TableView], but may return null.
 ///
 /// Used by [TableCellBuilderDelegate.builder] to build cells on demand for the
 /// table.
-typedef TableViewCellBuilder = Widget? Function(
+typedef TableViewCellBuilder = TableViewCell Function(
```

Comment:
This felt like a bug I came across. Typically in Flutter scrolling land, returning null from the child delegate means we have reached the end of the children. This is not currently supported, but is part of supporting infinite children in the table: https://github.com/flutter/flutter/issues/131226.

I think it was here by mistake since if you return null currently, with or without this change, it will crash! :(